jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/configuration/ResourceErrorHandlingRunSteps.java

```diff
@@ -30,7 +30,7 @@ public void deleteResourceEditorText(final String text) {
         final String normalizedText = text.charAt(0) == '"' ? text.substring(1, text.length() - 1) : text;
         By by = By.xpath("(//pre[contains(@class, 'CodeMirror-line')]//span[text()='" + normalizedText + "'])[1]");
         jwalaUi.click(by);
-        jwalaUi.doubleClick();
+        jwalaUi.backSpace();
```

Comment:
Are we sending the back space twice now?

Reply:
Here we are calling the same method but for different use case. Actually, we do delete "{" tag by selecting it with double click and send-keys over it. With new capabilities provided it was not functioning well. So, i removed the double click click functionality with backspace function + send keys which is exactly required for the operation.